static/scss/comment.scss

```diff
@@ -166,15 +166,15 @@ $replyPaddingLeftPhone: 10px;
 
       .authored-date {
         font-size: 14px;
-        color: $navy;
+        color: $gray69;
```

Comment:
looking at the `_variables.scss` file (and sorry I didn't realize this before) we already have a `#b0b0b0` color defined, as `$font-grey-light`

Reply:
thanks, somehow I missed that (probably searched for 0b0b0b instead of b0b0b0, I bet...doh).